astropy/coordinates/tests/test_sky_coord.py

```diff
@@ -1551,3 +1552,31 @@ class MockHeliographicStonyhurst(BaseCoordinateFrame):
 
     fr = MockHeliographicStonyhurst(lon=1*u.deg, lat=2*u.deg, radius=10*u.au)
     SkyCoord(0*u.deg, fr.lat, fr.radius, frame=fr) # this was the failure
+
+
+def test_multiple_aliases():
+    # Define a frame with multiple aliases
+    class MultipleAliasesFrame(BaseCoordinateFrame):
+        name = ['alias_1', 'alias_2']
+        default_representation = SphericalRepresentation
+
+    # Register a transform, which adds the aliases to the transform graph
+    tfun = lambda c, f: f.__class__(lon=c.lon, lat=c.lat)
+    ftrans = FunctionTransform(tfun, MultipleAliasesFrame, MultipleAliasesFrame,
+                               register_graph=frame_transform_graph)
+
+    coord = SkyCoord(lon=1*u.deg, lat=2*u.deg, frame=MultipleAliasesFrame)
+
+    # Test attribute-style access
+    assert coord.alias_1 == coord
```

Comment:
Here, maybe `is` is appropriate.

Reply:
Ah, good catch.  I've changed both checks to `is`.